### PR TITLE
Ryan Donovan

### PR DESCRIPTION
**Pull request by Ryan-Donovan33**
@mtrew2015

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -34,14 +34,25 @@ Edit this document to include your answers after each question. Make sure to lea
...
+
 4. While using flexbox, what axis does the following property work on: ```align-items: center```?
 
+    The cross-axis
+
 5. Explain why git is valuable to a team of developers.
 
+    Able to keep track of old versions of code, able to see who made the changes, and a nice way to log your code.
+
 You are expected to be able to answer all these questions. Your responses contribute to your Sprint Challenge grade. Skipping this section *will* prevent you from passing this challenge.
```
Comment (CodeDiversity) [inline README.md:57]: Great answers!

File: css/index.css [hunk 2]
```diff
@@ -48,99 +129,250 @@ table {
...
+	align-content: flex-end;
+	border-bottom: 2px dotted black;
+}
+
+.middle-section div img {
+	width: 100%;
+	height: auto;
+	padding: 5px;
+}
+
```
Comment (CodeDiversity) [inline css/index.css:301]: Great class names.

File: css/index.css [hunk 3]
```diff
@@ -48,99 +129,250 @@ table {
...
+/* Stretch Goal CSS below */
+
+.blog {
+	display: flex;
+	flex-flow: row wrap;
+}
+
+.blog div {
+	width: 44%;
+	display: block;
```
Comment (CodeDiversity) [inline css/index.css:374]: Awesome job nailing the stretch goal.

File: index.html [hunk 4]
```diff
@@ -30,16 +45,16 @@ <h2>The Past</h2>
...
-                <div class="box">Box 4</div>
-                <div class="box">Box 5</div>
-                <div class="box">Box 6</div>
-                <div class="box">Box 7</div>
-                <div class="box">Box 8</div>
-                <div class="box">Box 9</div>
-                <div class="box">Box 10</div>
+                <div class="box box1">Box 1</div>
+                <div class="box box2">Box 2</div>
+                <div class="box box3">Box 3</div>
```
Comment (CodeDiversity) [inline index.html:50]: well done,

File: css/index.css [hunk 5]
```diff
@@ -48,99 +129,250 @@ table {
...
+}
+
+.navigation {
+	display: flex;
+	width: 100%;
+	justify-content: space-around;
+	align-items: center;
+	margin: 5px;
+}
+
```
Comment (CodeDiversity) [inline css/index.css:167]: Nice job with the navigation, consider the spacing on the design file, perhaps a bit more margin on top would help here.

**Review by CodeDiversity — APPROVED** [review]
Overall, great job you really demonstrated a strong understanding of the knowledge from this Sprint and you completed a stretch goal. Keep up the great work!

**Comment by mtrew2015** [comment]
Don't forget to add me as a contributor, so I can merge the PR.

**Pull request merged**